* Meson below 0.48.0 chokes on duplicate identical build_rpath arguments.
* The RPATH hackery is pointless, as Meson nowdays includes all
  dependencies' paths in the transient build-tree RPATHs anyhow.

--- a/tools/meson.build
+++ b/tools/meson.build
@@ -12,10 +12,6 @@
   output : 'Bam2SamVersion.h',
   configuration : pbbam_Bam2SamVersion_h_config)
 
-if pbbam_htslib_dep.found()
-  pbbam_htslib_libdir = pbbam_htslib_dep.get_pkgconfig_variable('libdir')
-endif
-
 pbbam_bam2sam_cpp_sources = [pbbam_Bam2SamVersion_h]
 pbbam_bam2sam_cpp_sources += files([
   'common/OptionParser.cpp',
@@ -28,7 +24,6 @@
   dependencies : [pbbam_htslib_dep, pbbam_thread_dep, pbbam_zlib_dep],
   include_directories : [pbbam_include_directories, include_directories('bam2sam')],
   install : get_option('build-tools'),
-  build_rpath : pbbam_htslib_libdir,
   cpp_args : pbbam_warning_flags)
 
 # tests
